src/Centreon/Domain/Contact/Contact.php

```diff
@@ -444,6 +446,7 @@ public function setAccessToApiRealTime(bool $hasAccessToApiRealTime): self
      */
     public function hasRole(string $role): bool
     {
+        $roles = $this->$roles;
```

Comment:
```suggestion
```

Reply:
Yes sorry it was part of my debugging :S